include/oboe/Definitions.h

```diff
@@ -485,6 +485,162 @@ namespace oboe {
       Stereo = 2,
     };
 
+    /**
+     * The channel mask of the audio stream. The underlying type is `uint32_t`.
+     * Use of this enum is convenient
+     */
+    enum class ChannelMask : uint32_t { // aaudio_channel_mask_t
+        /**
+         * Audio channel mask is not specified.
+         */
+        Unspecified = kUnspecified,
+
+        /**
+         * Channel position masks. Use the combinations of the channel position masks.
+         */
+        FRONT_LEFT = 1 << 0,
```

Comment:
For consistency, these C++ enums should be Capitalized, eg. FrontLeft, LowFrequency, etc.

Reply:
I originally thought of keeping them consistent. But I finally make the single channel mask different from the combo ones as the combo ones are more designed for app usage. I was thinking of distinguish them as the using the single channel mask may be rejected by aaudio. Do you think it will be better to keep them consistent?

Reply:
Would it be possible to use private enum classes if these are not intended to be used by developers?

Reply:
We actually allow use those enums by developers if they want to make their combo in aaudio. But their combo will be rejected if it is not the ones known by aaudio.

BTW, I don't have an idea about how to make these enum private and only be used by oboe library.

Reply:
If we are allowing use of these, we should make them consistent. Maybe we can put a disclaimer with comments